sway-book/src/documentation/language/functions/return.md

```diff
@@ -0,0 +1,39 @@
+# Returning from functions
+
+In the previous sections we have seen how functions return values without much information. In this section we will take a closer look at how we can return data from a function.
```

Comment:
unclear what "without much information" refers to here. I _think_ you mean that we've seen it happen but haven't discussed it in much detail. But it could also be taken to mean that the return values didn't contain much information.